packages/foreman/foreman/foreman.spec

```diff
@@ -147,9 +148,10 @@ Requires: %{?scl_prefix}rubygem(rack-jsonp)
 
 # Build dependencies
 %{?systemd_requires}
-BuildRequires: gettext
 BuildRequires: asciidoc
+BuildRequires: gettext
 BuildRequires: %{scl_ruby_bin}
+BuildRequires: %{?scl_prefix_ruby}rubygem(bigdecimal)
```

Comment:
This one is a bit odd - the other bigdecimal was in the managed section, this one isn't. Below is `# start specfile main BuildRequires` which is the managed section.

Reply:
It is a bit odd, seems like the packaged Ruby on EL7 requires bigdecimal and on EL8 they broke it out to not be required by default.